packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -322,7 +329,15 @@ export const Stackbox = () => {
             </div>
             <div className="space-y-1">
               <div className="flex flex-col border divide-y md:divide-y-0 md:flex-row rounded-2xl border-surface-50 md:divide-x divide-surface-50">
-                <div className="flex flex-col w-full p-3 space-y-2 hover:bg-surface-25">
+                <div
+                  className={cx(
+                    "flex flex-col w-full p-3 space-y-2 hover:bg-surface-25",
+                    {
+                      "border border-danger-500 rounded-l-2xl":
```

Comment:
Small detail: I would change the danger color to 200, to not be so strong. 
```suggestion
                      "border border-danger-200 rounded-l-2xl":
```